app/views/welcomes/_footer.haml

```diff
@@ -0,0 +1,6 @@
+%footer
+  .container
+    .row
+      .col-12
+        %h6= "&copy; #{year} INSTEDD ILAB SOUTHEAST ASIA. ALL RIGHTS RESERVED.".html_safe
```

Comment:
@radinreth normally copyrights or all rights reserved stated on the year born(static) and it's not dynamically following the current date

Reply:
ok, I will set it to static